man/docker-inspect.1.md

```diff
@@ -194,7 +194,7 @@ To get information on a container use its ID or instance name:
 
 To get the IP address of a container use:
 
-    $ docker inspect --format='{{.NetworkSettings.IPAddress}}' d2cc496561d6
+    $ docker inspect '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' d2cc496561d6
```

Comment:
Perhaps use the other format here as well

Also we may want to mention this is the IP address of the bridge network; there could be multiple networks